src/isAbsolute.ts

```diff
@@ -0,0 +1,16 @@
+import * as URI from 'urijs';
+import { startsWithWindowsDrive } from './startsWithWindowsDrive';
+
+const PATH_SEPARATOR_REGEXP = /[/\\]/;
+
+// based on https://github.com/nodejs/node/blob/9946c597077070941309e5b2986a810e52978a97/lib/path.js#L349
+export function isAbsolute(filepath: string) {
```

Comment:
What about using the `isUrl` helper, and use URI if it's a url, otherwise the other checks in here that target file paths?